SteamProject/MySteam/Game.h

```diff
@@ -8,28 +8,34 @@ class FGame
     int ReleaseDay;
     int ReleaseMonth;
     int ReleaseYear;
-    
+
 public:
 
     FGame();
 
-    FGame(const std::string NewGameName, const std::string NewStudioName, const int NewReleaseDay, const int NewReleaseMonth, const int NewReleaseYear);
+    FGame(const std::string& NewGameName, const std::string& NewStudioName, const int& NewReleaseDay, const int& NewReleaseMonth, const int& NewReleaseYear);
```

Comment:
References are useful to avoid potentially hard-to-copy types. This will never be the case with built-in types, so for built-in types, it's better to pass them by value.